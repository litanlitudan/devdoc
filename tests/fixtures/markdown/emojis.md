# :fire: Emoji!

:woman_technologist:
:joystick:
:zap:
:link:
:cupid: