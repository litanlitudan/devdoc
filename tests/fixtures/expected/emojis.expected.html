<h1 id="fire-emoji">🔥 Emoji!</h1>
<p>👩‍💻
🕹
⚡️
🔗
💘</p>
